app/Notifications/Training/TrainingPlaceOffer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Notifications\Training;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class TrainingPlaceOffer extends Notification
```

Comment:
Might be worth putting a TODO in this class just so its super obvious it isn't ready

Reply:
done